ci-operator/config/red-hat-data-services/ods-ci/red-hat-data-services-ods-ci-master__rhoai2.13-ocp4.16-lp-interop.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  rhoai-tests:
+    name: ods-ci
+    namespace: ci
+    tag: dev
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhoai-interop-aws
+  cron: 0 0 11 6 *
+  steps:
+    cluster_profile: ibmcloud-cspi-qe
```

Comment:
Should we create our own cluster profile to IBM?

Reply:
Yes. We may also create cluster profile on Open Stack for testing.